packages/react-core/src/demos/DescriptionList/DescriptionList.md

```diff
@@ -0,0 +1,25 @@
+---
+id: Description list
+section: components
+---
+
+import BarsIcon from '@patternfly/react-icons/dist/esm/icons/bars-icon';
```

Comment:
The imports here just need to be cleaned up to remove things that aren't actually being used in this demo.